Python/bltinmodule.c

```diff
@@ -1411,6 +1428,7 @@ map_traverse(mapobject *lz, visitproc visit, void *arg)
 static PyObject *
 map_next(mapobject *lz)
 {
+    Py_ssize_t i; // Needed for error reporting
```

Comment:
```suggestion
    Py_ssize_t i;
```

Moreover, `i` is now shadowed in L1466-1468.